provider/gke/gke.go

```diff
@@ -215,9 +215,15 @@ func (c *GKE) NodePoolCreate(*kingpin.ParseContext) error {
 				NodePool:  node,
 			}
 			log.Printf("Cluster nodepool create request: cluster '%v', nodepool '%v' , project `%s`,zone `%s`", reqN.ClusterId, reqN.NodePool.Name, reqN.ProjectId, reqN.Zone)
-			_, err := c.clientGKE.CreateNodePool(c.ctx, reqN)
+
+			err := provider.RetryUntilTrue(
+				fmt.Sprintf("creating operation to create nodepool:%v", reqN.NodePool.Name),
```

Comment:
I did not want to keep it `nodepool creation` because when this returns successfully, it will print `nodepool creation` done. Then `nodePoolRunning()` will start.

It will print `creating nodepool:prometheus-84 Retrying after 10 seconds`.
This can easily become confusing

Reply:
good point , in that case can change the `nodePoolRunning` message to
`fmt.Sprintf("checking nodepool running status for:%v", reqN.NodePool.Name)`